API Blueprint Specification.md

```diff
@@ -76,10 +83,6 @@ All of the blueprint sections are optional. However, when present, a section **m
     + [`0-1` **URI Parameters** section](#def-uriparameters-section)
     + [`0-1` **Attributes** section](#def-attributes-section)    
     + [`0-1` **Model** section](#def-model-section)
-        + [`0-1` **Headers** section](#def-headers-section)
```

Comment:
@adams-sarah any reason you took these out from here and left them for request / response sections below?